src/States/MapViewStateDraw.cpp

```diff
@@ -30,31 +30,29 @@ extern std::vector<void*> path;
 
 
 namespace {
-	Timer glowTimer;
-	const int GlowStepSize = 20;
-
-	int glowStep;
-	int glowStepDirection = 1;
-
-
-	void updateGlowTimer()
+	uint8_t getGlowIntensity()
```

Comment:
Consider 'calcGlowIntensity` because this is doing more than just "getting" a value.